### PR TITLE
chore(blog): deleting old banner image from blog

### PR DESCRIPTION
**Pull request by amyshwang**
I noted that the preview image for "[Comparing InfluxDB and QuestDB Databases](https://questdb.io/blog/2021/11/29/questdb-versus-influxdb/)" is wrong; it is not using `banner.webp`. I deleted the thumbnail image generated using the old banner - to check if this works.

### CODE REVIEW

**Comment by amyshwang** [comment]
Problem:
Old banner image displayed:
Screenshot 2023-05-31 at 18 04 37

My fix created this:

Screenshot 2023-05-31 at 18 04 03

I am not sure how to fix this now...

**Comment by argshook** [comment]
hey @amyshwang i fixed the issue by:
1. manually running `node scripts/optimise.js static/img/blog/2021-11-29/banner.webp`
2. this optimised (unnecesarily though) the `banner.webp` but in addition it created `banner.thumb.webp` which was missing

in addition i added `git add` to make sure the next time some `banner.webp` is added (or changed), the generated thumbnail will be added to git staging and later comitted.

In other words: the issue you had should no longer repeat.

**Review by argshook — APPROVED** [review]
